src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_on_unicast_transports.cs

```diff
@@ -71,7 +71,8 @@ public class SubscriberA : EndpointConfigurationBuilder
         {
             public SubscriberA()
             {
-                EndpointSetup<DefaultServer>(c => { c.UnicastRouting().AddPublisher(PublisherEndpoint, typeof(MyEvent)); });
+                //EndpointSetup<DefaultServer>(c => { c.Routing().AddPublisher(PublisherEndpoint, typeof(MyEvent)); });
+                EndpointSetup<DefaultServer>(c => { c.UseTransport<MsmqTransport>().RegisterPublisherForType(PublisherEndpoint, typeof(MyEvent)); });
```

Comment:
I wonder whether this workaround is good enough for us to go ahead with the new api. Since it will only be AT which need a way to work around the `UseTransport` API I guess we could live with that.